howso/utilities/feature_attributes/pandas.py

```diff
@@ -472,19 +484,46 @@ def _as_float(value, dtype):
                                 elif actual_max_f == mode_f:
                                     max_f = actual_max_f
 
-                output = {'min': min_f, 'max': max_f, 'allow_null': allow_null}
+                output = {
+                    'min': min_f, 'max': max_f,
+                    'observed_min': actual_min_f, 'observed_max': actual_max_f,
+                    'allow_null': allow_null,
+                }
             else:
                 # If no min/max were found from the data, use min/max size of
                 # the data type.
                 min_value, max_value = self._get_min_max_number_size_bounds(
                     feature_attributes, feature_name)
                 if min_value is not None and max_value is not None:
-                    output = {'min': min_value, 'max': max_value, 'allow_null': allow_null}
+                    output = {
+                        'min': min_value, 'max': max_value,
+                        'observed_min': min_f, 'observed_max': max_f,
```

Comment:
Wont the min_f/max_f be potentially NaN here? Should we include the observed value in that case? Or should it just use min/max_value

Reply:
Fair point. Will update.